src/OrbitGl/CaptureViewElement.h

```diff
@@ -35,6 +35,11 @@ class CaptureViewElement : public Pickable {
   void SetSize(float width, float height) { size_ = Vec2(width, height); }
   [[nodiscard]] Vec2 GetSize() const { return size_; }
 
+  [[nodiscard]] orbit_accessibility::AccessibleInterface* GetOrCreateAccessibleInterface();
+  [[nodiscard]] const orbit_accessibility::AccessibleInterface* GetAccessibleInterface() const {
+    return accessible_interface_.get();
+  }
```

Comment:
Exactly, it's only for the `const` case. Accessible interfaces may need this to retrieve the parent-interface in `AccessibleParent`, which is const and can safely assume that the parent has already created the interface since the tree is always traversed from the root. And in this case, it also needs to be public.